nhais-adaptor/incoming.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+import adaptor.incoming.operation_definition_adaptor as adaptor
+import edifact.incoming.parser.deserialiser as deserialiser
+from PyInquirer import prompt
+
+dirs = os.listdir("./mailbox/NHAIS")
+
+nhais_outbox_choices = [
+    {
+        'type': 'list',
+        'name': 'nhais',
+        'message': 'Select a NHAIS outbox to read from: ',
+        'choices': dirs
+    }
+]
+
+answers = prompt(nhais_outbox_choices)
+nhais_cypher = answers["nhais"]
+
+with open(f"./mailbox/NHAIS/{nhais_cypher}/outbox/edifact.txt", "r") as incoming_edifact_file:
+    incoming_interchange_raw = incoming_edifact_file.read()

```

Comment:
There's a method in the utilities for loading a string from a file, might also be worth using the Path() module to manage your file path strings as I don't think they convert to Unix very nicely otherwise.